src/components/input/keyboardnavigation.js

```diff
@@ -145,6 +145,32 @@ define(["inputManager", "layoutManager"], function (inputManager, layoutManager)
             }
         });
     }
+    // For hiding the mouse while playbackç
+    require(["components/input/mouseManager"]);
+
+    // Gamepad initialization. No script is required if no gamepads are at boot time, saving a bit of resources._10k
+    // Whenever the gamepad is connected, we hand all the control of the gamepad to gamepadtokey.js by removing the event handler
+    function isGamepadConnected() {
+        var gamepads = navigator.getGamepads();
+        for (var i = 0, len = gamepads.length; i < len; i++) {
+            var gamepad = gamepads[i];
+            if (gamepad) {
+                return true;
+            }
+        }
+        return false;
+    }
+
+    function attachGamepadScript(e) {
+        if (isGamepadConnected()) {
```

Comment:
why do we want to check that, if that is fired on the `gamepadconnected` event?

Reply:
True, thanks!